README.md

```diff
@@ -524,15 +524,17 @@ strategies:
 
 This strategy evicts pods that are older than `maxPodLifeTimeSeconds`.
 
-You can also specify `podStatusPhases` to `only` evict pods with specific `StatusPhases`, currently this parameter is limited
-to `Running` and `Pending`.
+You can also specify `states` parameter to **only** evict pods matching the following conditions:
```

Comment:
@a7i the latest release (0.24.1) doesn't actually support the `states` parameter, and silently ignores it if it's specified. Someone reading these instructions who installs 0.24.1 and enables the `PodLifeTime` policy with `states` other than `Running` will be unpleasantly surprised when their `Running` pods are terminated... 

I'd recommend clarifying the instructions to specify which versions accept the `state` parameter. I'd also suggest reconsidering whether `Running` pods should be targeted by default; it would be more reasonable (IMO) to require `states` so there's no room for surprises like this.

Reply:
Hi @blampe thanks for the feedback.

Looking at the [0.24.1 README](https://github.com/kubernetes-sigs/descheduler/tree/v0.24.1), I do not see `states` as a configuration so I assume you're suggesting that regardless of which README you're looking at (main branch or a tag), the policy description should specify the minimum version a field is supported in. As an alternative, what is your feedback on [fail fast validation](https://github.com/kubernetes-sigs/descheduler/pull/706) where "erroneous" fields cause Descheduler to fail?

As far as excluding `Running` from `states`, this would be a breaking change. Would it be sufficient to update the documentation to indicate that ALL pods regardless of state/status are candidates for eviction if `states` / `podStatusPhases` is not specified?

Reply:
https://github.com/kubernetes-sigs/descheduler/pull/928